src/coord/axisTickLabelBuilder.ts

```diff
@@ -132,8 +132,10 @@ export function createAxisLabels(axis: Axis, ctx: AxisLabelsComputingContext): {
         return {
             labels: zrUtil.map(ticks, numval => {
                 const tick = {value: numval};
+                const index = ticks.indexOf(numval);
```

Comment:
@Ovilia Currently, using `indexOf` to retrieve the tick index is not necessary and may lead to a performance issue. The index can be obtained from the second callback parameter of `zrUtil.map`.

```js
zrUtil.map(ticks, (numval, index) => {})
```